example/template.env

```diff
@@ -1,55 +1,57 @@
-## Example / Template .env file for Wikibase release pipeline docker compose example
+# ============================================================================================
+# BASIC CONFIGURATION: Review and update entries in this section to get started
+# ============================================================================================
+
 # WARNING: Do not add comments on the same line as env vars, as in some environments they will be included in the var!
 
-## Image Configuration
-WIKIBASE_IMAGE_NAME=wikibase/wikibase:1.41.1-wmde.20
-WDQS_IMAGE_NAME=wikibase/wdqs:0.3.137-wmde.20
-WDQS_FRONTEND_IMAGE_NAME=wikibase/wdqs-frontend:wmde.20
-ELASTICSEARCH_IMAGE_NAME=wikibase/elasticsearch:7.10.2-wmde.20
-WIKIBASE_BUNDLE_IMAGE_NAME=wikibase/wikibase-bundle:1.41.1-wmde.20
-QUICKSTATEMENTS_IMAGE_NAME=wikibase/quickstatements:wmde.20
-WDQS_PROXY_IMAGE_NAME=wikibase/wdqs-proxy:wmde.20
-MYSQL_IMAGE_NAME=mariadb:10.11
-
-## Mediawiki Configuration
-## Admin password
-## Passwords must be at least 10 characters.
-## Your password must be different from your username.
-## Your password must not appear within your username.
-## The password must not be in a list of very commonly used passwords. Please choose a unique password.
-MW_ADMIN_PASS=change-this-password
+# MediaWiki configuration
 MW_ADMIN_NAME=admin
 MW_ADMIN_EMAIL=admin@example.com
+# Passwords must be at least 10 characters
+# Your password must be different from your username
+# Your password must not appear within your username
+# The password must not be in a list of very commonly used passwords
+MW_ADMIN_PASS=change-this-password
+# Should be unique across all instances of this MediaWiki,
+# see https://www.mediawiki.org/wiki/Manual:$wgSecretKey
 MW_SECRET_KEY=some-secret-key
-MW_WG_ENABLE_UPLOADS=false
 
-## Jobrunner Configuration
-MAX_JOBS=1
-
-## Database Configuration
+# Database configuration
 DB_NAME=my_wiki
 DB_USER=sqluser
-DB_PASS=change-this-sqlpassword
+DB_PASS=change-this-password
+
+# The publically accessible hostnames which point to this host. You can instead just use the ports to access the services,
+# but without valid publically acceessible addresses QuickStatements will not be able to OAuth with Wikibase.
+WIKIBASE_PUBLIC_HOST=wikibase.local.gd
+WDQS_FRONTEND_PUBLIC_HOST=wdqs-frontend.local.gd
+QUICKSTATEMENTS_PUBLIC_HOST=quickstatements.local.gd
 
-## Wikibase Configuration
+# Ports published on host for front-end services. These probably only need to be changed if
```

Comment:
Change to "The port numbers used for the front-end services. These can be reset to any unique set of port numbers that don't conflict with ports already being used on the host. Port 80 and 443 are used by the nginx-proxy reverse proxy service if that is not disabled. You can access the respective services directly through these ports (e.g. http://localhost:8880) on the host machine, or through the proxied public addresses above if configured."